plugins/ruby-foreman-hdm/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: ruby-foreman-hdm
+Section: ruby
+Priority: optional
+Maintainer: Tim <tim@bastelfreak.de>
+Build-Depends: debhelper-compat (= 13),
+               gem2deb (>= 1),
+               rake,
+               foreman-assets(>= 3.5.0~rc1),
+               foreman-nulldb (>= 3.5.0~rc1)
```

Comment:
https://github.com/betadots/foreman_hdm/blob/db90c949b510bde90e8387935791292296db8c4b/lib/foreman_hdm/engine.rb#L21 says 3.4+ -- what's correct? :)

(not that it matters too much, but seems odd)